### DISCUSSION TITLE
Image background

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/GoOdf#95] [General]
Hi,
When I choose my own panel background image, I select an image as my screen resolution 1280*1024. 

In GOodf the panel looks good but when I open it in GrandOrgue, background is shortened and not fill all the screen .. how can I fix it ?

Many thanks !

**Comment by larspalo** [comment]
@Jean-Gabriel1982 Let me guess that it's because GrandOrgue always provide the title bar that uses up some of the available height, and then some scaling is used to adapt the panel to available size. In GoOdf I don't use any scaling at all so the panel view will always be at 100% which always makes the layout exact.

Currently there's no full screen option for GO panels so there will always be some loss of available height (and possibly width). Exaxtly how much the window manager will consume is not easy to tell. Your best bet is to experiment and see what is the max size possible before any scaling is enforced. Also, that result would be highly dependent on your screen size and used window manager and would likely differ from what someone else uses.

When testing for maximum possible size with GoOdf pay close attention to if any part of the window is outside the screen. I've not tested exactly what the max height is on my Ubuntu 20.04 but it seems like about 24 px is lost to the system task bar, then another maybe 38 px is lost to the window title bar which would reduce the max height by about 62 px. Most likely a few px is also lost on the width.

So for my 1920x1080 screen, maybe something like 1918x1018 is perhaps the largest panel size I can get away with using before some kind of scaling kicks in. You'll have to experiment with what you can use yourself, the layout of GoOdf might not be 100% accurate to what GO would do within 100% scaling but I try to improve it as soon as I detect differences.

**Reply by Jean-Gabriel1982** [reply]
Thank you very much, Lars, for all these explanations. I understand better now.
In fact, my panel is “bigger” on GOodf than on GrandOrgue!

On GrandOrgue, it is in fact shrunk, which allows, on the one hand, to better see certain elements which could have been a little too close to the edges on GOodf.

The downside is these two black bands on the sides which don't look very good.
A few weeks ago, I asked if it was possible to have the full screen on GrandOrgue. Is this something that could be done in the near future? That would be really, really great!

**Comment by larspalo** [comment]
>  Is this something that could be done in the near future? That would be really, really great!

I cannot guarantee anything like that even though I think it could be useful in some sense. I'm personally much more interested in working on the audio qualities of GO like the Swellbox modeling, Tremulant model improvement and perhaps a Wind model than the graphical aspects.

**Reply by Jean-Gabriel1982** [reply]
Of course this is more important ...
